projects/ad4630_fmc/Readme.md

```diff
@@ -1,8 +1,7 @@
-# AD4630-FMC HDL Project
+# AD4630-24 Eval FMC HDL Project
 
 Here are some pointers to help you:
-  * [Board Product Page](https://www.analog.com)
```

Comment:
Only the link part of this line should be empty if there is no link for board page. The " * [Board Product Page]" text should remain there.

Reply:
Done